regolith/helpers/a_presentationhelper.py

```diff
@@ -0,0 +1,148 @@
+"""Helper for adding a presentation to the presentation collection.
+"""
+import datetime as dt
+import dateutil.parser as date_parser
+from dateutil.relativedelta import relativedelta
+import sys
+
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import (
+    all_docs_from_collection,
+    get_pi_id,
+)
+
+TARGET_COLL = "presentations"
+ALLOWED_TYPES = ["award", "keynote", "plenary", "invited", "contributed_oral",
+                 "poster", "colloquium", "seminar","other"]
+ALLOWED_STATI = ["accepted"]
+
+
+def subparser(subpi):
+    subpi.add_argument("--database",
+                       help="The database that will be updated.  Defaults to "
+                            "first database in the regolithrc.json file."
+                       )
+    subpi.add_argument("place", help="the place of the presentation, location if conference,"
+                                     "institution for seminars"
+                       )
+    subpi.add_argument("name", help="name of the presentation, meeting name if meeting,"
+                                    "department if seminar",
+                       )
+    subpi.add_argument("-t","--type", help=f"types, from {ALLOWED_TYPES}. Default is other",
+                       default="invited"
+                       )
+    subpi.add_argument("-d", "--begin_date",
+                       help="Input begin date for this expense. "
+                            "In YYYY-MM-DD format. Defaults to today's date",
+                       )
+    subpi.add_argument("-e,", "--end_date",
+                       help="Input end date for this expense. "
+                            "In YYYY-MM-DD format. Defaults to today's date",
+                       )
+    subpi.add_argument("-p", "--person",
+                       help="the person submitting the presentation, used for presentation name",
+                       default="sbillinge"
+                       )
+    subpi.add_argument("-a", "--abstract",
+                       help="abstract description, defaults to tbd",
+                       default='tbd'
+                       ) #isn't an abstract a description?
+    subpi.add_argument("-s", "--status",
+                       help="status of the presentation, default is accepted",
+                       default="accepted"
+                       )
+    subpi.add_argument("-x", "--title",
+                       help="the title of the presentation, default is tbd",
+                       default='tbd'
+                       )
+    subpi.add_argument("-y", "--authors", nargs="+",
```

Comment:
don't use ``-x`` and ``-y`` for shortcuts but something more relevant.  Maybe -t for title and -y for type? -u for author?

Please put the optional args in the order of importance and move --database to the bottom.  Glancing at the optional args, personally I would rarely enter authors, type, status or person as the defaults are good, but title I probably would.  

Isn't grant a thing that is added to presentation?

Reply:
I do not think grants are added in presentations, because they are not present in schemas, and they are only used for expenses in new presentation.py.